### ISSUE TITLE
jaxlib source code releases

### ISSUE THREAD

**Issue by adamjstewart** [jax-ml/jax#24137]
I'm trying to build JAX from source but I noticed that there are no source code releases for jaxlib 0.4.33 or 0.4.34 on GitHub or PyPI. Are there plans to publish the source code for those releases, or should we instead use the JAX release tarballs?

P.S. I'm packaging JAX in a package manager, I can't just use the PyPI wheels.

**Comment by hawkinsp** [comment]
Correct. We're now using the `jax` tag to build and release both. Since we now much more closely tie the versions of the two, it seemed redundant to have two tags.

It's possible in the future we may still release `jax` (the Python bits) without releasing `jaxlib` (the C++ bits), but we will never release `jaxlib` without also releasing `jax`. It should also always be the case that whenever `jaxlib` is released, it is released from the same tag as `jax`.

Hope that helps!
